addon/-private/compatibility.js

```diff
@@ -0,0 +1,15 @@
+import require from 'require';
+
+function _hasLegacyEmberTestHelpers() {
+  return require.has('ember-test-helpers');
+}
+
+function _hasLatestEmberTestHelpers() {
+  return require.has('@ember/test-helpers');
+}
+
+if (!_hasLegacyEmberTestHelpers() && !_hasLatestEmberTestHelpers()) {
+  throw new Error('ember-test-helpers or @ember/test-helpers must be installed');
+}
+
+export { default as wait } from 'ember-test-helpers/wait';
```

Comment:
Maybe this is too nitpicky and not a case we care about, but this will throw in a technically valid scenario. If the user is only using `ember-cli-page-object` for acceptance tests, then we actually don't need `ember-test-helpers` or `@ember/test-helpers` as the acceptance context uses `window.wait()` which comes from Ember itself.

If that is a case we want to continue supporting, then my suggestion would be to export a function that runs the checks before calling through to `wait`, so we only check and assert when the `wait` functionality is being used, rather than when the modules are loading. Or run the checks at module scope and store the result in a variable, and then check and throw inside the exported function if we're worried about the perf implications of running `require.has()` twice on every `wait()` call.